roles/tackle/tasks/main.yml

```diff
@@ -410,3 +467,30 @@
     state: present
     definition: "{{ lookup('template', 'networkpolicy.yml.j2') }}"
   when: feature_isolate_namespace|bool
+
+- when:
+    - not(feature_auth_required|bool) or not(feature_auth_type == "keycloak")
+  block:
+    - name: "Deprovision RHSSO Keycloak CR"
```

Comment:
Does this by itself not remove everything?

Reply:
This block does. It removes the keycloak configuration, the SSO deployment, and the DB deployment. I left the PVC alone so we don't destroy data. If someone wanted to try Oauth or switch to auth false for some reason and then decides to switch back their configured users will be as they left them, etc.